example/volume_Mount/redis-cluster.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: redis.redis.opstreelabs.in/v1beta1
+kind: RedisCluster
+metadata:
+  name: redis-cluster
+spec:
+  clusterSize: 3
+  clusterVersion: v7
+  persistenceEnabled: true
+  securityContext:
+    runAsUser: 1000
+    fsGroup: 1000
+  kubernetesConfig:
+    image: quay.io/opstree/redis:v7.0.5
+    imagePullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 101m
+        memory: 128Mi
+      limits:
+        cpu: 101m
+        memory: 128Mi
+    redisSecret:
+      name: redis-secret
+      key: password
+    # imagePullSecrets:
+    #   - name: regcred
+  redisExporter:
+    enabled: false
+    image: quay.io/opstree/redis-exporter:v1.44.0
+    imagePullPolicy: Always
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
+      limits:
+        cpu: 100m
+        memory: 128Mi
+# Environment Variables for Redis Exporter
+    # env:
+    # - name: REDIS_EXPORTER_INCL_SYSTEM_METRICS
+    #   value: "true"
+    # - name: UI_PROPERTIES_FILE_NAME
+    #   valueFrom:
+    #     configMapKeyRef:
+    #       name: game-demo
+    #       key: ui_properties_file_name
+    # - name: SECRET_USERNAME
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: mysecret
+    #       key: username
+#  redisLeader:
+#    redisConfig:
+#      additionalRedisConfig: redis-external-config
+#  redisFollower:
+#    redisConfig:
+#      additionalRedisConfig: redis-external-config
+  storage:
+    volumeClaimTemplate:
+      spec:
+        # storageClassName: standard
+        accessModes: ["ReadWriteOnce"]
+        resources:
+          requests:
+            storage: 1Gi
+    volumemount:
+      volume:
+        - name: example-config
+          configMap:
+            name: example-configmap
+      mount:
+        - mountPath: /config
+          name: example-config
+  # nodeSelector:
+  #   kubernetes.io/hostname: minikube
+  # priorityClassName:
+  # Affinity:
+  # Tolerations: []
```

Comment:
Can we remove other unnecessary maps and objects, just like in the other examples folder? Also, let's keep the name of folder in `snake_case`.

Reply:
Also, add one example for standalone as well.